### DISCUSSION TITLE
how to create scrollbar for legends [using ResponsiveLine]

### DISCUSSION THREAD

**Discussion by thekirankadam** [plouc/nivo#2281] [Q&A]
In my case, I have too many legends and they are getting trimmed or going out of allocated space. How to enable the scrollbar for legends. Also is there a way to text wrap legend as in the case of very long legend text it's also going out of scope

**Comment by plouc** [comment]
You cannot have scrollbars inside a SVG, this would have to be implemented in user space.

**Reply by thekirankadam** [reply]
The userspace scrollbar would not help if the legends itself are getting trimmed

**Comment by plouc** [comment]
And as for scrollbars, textwrap is not supported by default.
